### DISCUSSION TITLE
Dymension chain onboarding proposal

### DISCUSSION THREAD

**Discussion by shaolin-flow** [wormhole-foundation/wormhole#3721] [Gateway]
### Website

https://dymension.xyz/

### Github

dymensionxyz

### Blog

https://medium.com/@dymension

### Documentation

https://docs.dymension.xyz/

### Community stats

- 138,000 Followers on Twitter 
- 5,000 Followers on Telegram 
- 120,000 Users on Discord

### Short Summary

Dymension is a blockchain providing security, inter-operability, and liquidity for RollApps. RollApps are modular blockchains that can be deployed within a few minutes with instant access to liquidity. RollApps use external DA for data publication such as Celestia and Avail. 

### Why should Wormhole add this chain?

Dymension is an IBC enabled blockchain and via Dymension all RollApp are connected for transferring tokens. Guardian do not need to run a full node of Dymension or any RollApp. 

The current devnet (Froopyland) has thousands of users (https://portal.dymension.xyz/). This will gradually become the mainnet to be deployed in the near future. To kickstart the blockchain Dymension (through its own governance) will allocate DYM rewards (native protocol token) for users bridging assets including wrapped SOL via Wormhole. Additionally, due to the popularity of Dymension's Genesis Rolldrop there are many users that will want to bridge from Dymension to Solana and back.

### Technology and Features

- IBC rollups (Each RollApp on Dymension is connected to IBC via the Dymension blockchain)
- Fraud proofs (Fraud proofs are verified on the Dymension blockchain incase of an invalid state transition)
- AMM (Dymension hosts an embedded AMM to facilitate easy cross-chain transfers.
- eIBC (Dymension adds a market making mechanism to facilitate fast withdrawals from optimistic rollups over IBC)
- RDK (Dymension core team is continually building out the RollApp Development Kit (RDK) which allows developers to easily bootstrap RollApps.

### Native token details

Dymension's monetary policy: https://docs.dymension.xyz/learn/dymension/dym/supply

**Supply:** 
- Initial Supply: 1 Billion DYM
- Max issuance: 10%
- Min issuance: 1%

**Demand**
- Validating bridge fee from RollApp to Dymension
- Protocol swap fee on Dymension AMM
- RollApp dynamic bond (staking DYM for operating RollApps)

### Technical requirements for running a node

None - [chain] will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).

### IBC Relaying Strategy

We are working with P2P and Forbole to provide relayer services to the Dymension <> Gateway IBC channel. This service is reflected in the delegation program. They will immediately create the channels for testnet and once mainnet launches and we will integrate the front-end into the [Portal](https://portal.dymension.xyz/). We will want to execute a full end to end flow including transfer of SOL to the Dymension testnet ASAP.

**Comment by SimonKraus** [comment]
I am hereby voicing our support for the Dymension onboarding.
 - Staking Facilities 🚀
